airflow/www/views.py

```diff
@@ -28,46 +28,45 @@
 import traceback
 from collections import defaultdict
 from datetime import timedelta
-from urllib.parse import quote
 
 import markdown
 import pendulum
 import sqlalchemy as sqla
-from flask import (
-    redirect, request, Markup, Response, render_template,
-    make_response, flash, jsonify, url_for)
+from flask import (Markup, Response, flash, jsonify, make_response, redirect,
+                   render_template, request, url_for)
 from flask_appbuilder import BaseView, ModelView, expose, has_access
 from flask_appbuilder.actions import action
 from flask_appbuilder.models.sqla.filters import BaseFilter
 from flask_babel import lazy_gettext
 from pygments import highlight, lexers
 from pygments.formatters import HtmlFormatter
-from sqlalchemy import func, or_, desc, and_, union_all
+from sqlalchemy import and_, desc, func, or_, union_all
+from urllib.parse import quote
 from wtforms import SelectField, validators
 
 import airflow
 from airflow import configuration as conf
-from airflow import models, jobs
+from airflow import jobs, models
 from airflow import settings
-from airflow.api.common.experimental.mark_tasks import (set_dag_run_state_to_success,
-                                                        set_dag_run_state_to_failed)
-from airflow.models import Connection, DagModel, DagRun, errors, Log, SlaMiss, TaskFail, XCom
-from airflow.ti_deps.dep_context import DepContext, QUEUE_DEPS, SCHEDULER_DEPS
+from airflow._vendor import nvd3
+from airflow.api.common.experimental.mark_tasks import (set_dag_run_state_to_failed,
+                                                        set_dag_run_state_to_success)
+from airflow.models import Connection, DagModel, DagRun, Log, SlaMiss, TaskFail, XCom, \
+    errors
+from airflow.ti_deps.dep_context import DepContext, SCHEDULER_QUEUED_DEPS
 from airflow.utils import timezone
 from airflow.utils.dates import infer_time_unit, scale_time_units
-from airflow.utils.db import provide_session, create_session
+from airflow.utils.db import create_session, provide_session
 from airflow.utils.helpers import alchemy_to_dict, render_log_filename
 from airflow.utils.state import State
-from airflow._vendor import nvd3
 from airflow.www import utils as wwwutils
 from airflow.www.app import app, appbuilder
 from airflow.www.decorators import action_logging, gzipped, has_dag_access
-from airflow.www.forms import (DateTimeForm, DateTimeWithNumRunsForm,
-                               DateTimeWithNumRunsWithDagRunsForm,
-                               DagRunForm, ConnectionForm)
+from airflow.www.forms import (ConnectionForm, DagRunForm, DateTimeForm,
+                               DateTimeWithNumRunsForm,
+                               DateTimeWithNumRunsWithDagRunsForm)
```

Comment:
Could you avoid/remove the import ordering/format changes? Having these in here makes it much harder to cherry-pick fixes back to the 1-10 release branch.

Reply:
(And make it less likely to conflict too)